bookshop/sqlalchemy/model_to_dict.py

```diff
@@ -98,3 +95,20 @@ def _recurse_on_model_or_list(
             paths=next_paths,
             return_immediately=return_immediately,
         )
+
+
+def deep_merge(dict1, dict2):
+    for k in set(dict1.keys()).union(dict2.keys()):
+        if k in dict1 and k in dict2:
+            if isinstance(dict1[k], dict) and isinstance(dict2[k], dict):
+                yield (k, dict(deep_merge(dict1[k], dict2[k])))
+            elif isinstance(dict1[k], list) and isinstance(dict2[k], list):
+                yield (k, list(dict1[k] + dict2[k]))
+            elif isinstance(dict1[k], list) and not isinstance(dict2[k], list):
```

Comment:
Are these list merges definitely what we want?

Reply:
I can't think of a situation where we would want lists merged for model to dict.

Reply:
Agreed, removed!